package.json

```diff
@@ -22,13 +22,15 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
     "pg-format": "^1.0.4",
```

Comment:
pg format would count as a Dependency in my opinion, we need it for seeding which is universally used